src/database/adminData.js

```diff
@@ -0,0 +1,12 @@
+const admin = {
+    firstName: 'admin',
+    lastName: 'admin',
+    username: 'super-admin',
+    email: 'admin@barefootnomad.com',
+    password: 'Admin@2020',
+    gender: 'male',
+    address: 'kigali',
+    role: 'Super Administrator',
+    isVerified: true,  
+}; 
+export default admin;
```

Comment:
if this is a `script` as the TTL mentioned,  Ithink you should create a separate folder for them. for instance src/database/scrpts